src/OpenTelemetry/Logs/LogRecordExportProcessorOptions.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="LogRecordExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Logs
+{
+    public class LogRecordExportProcessorOptions
```

Comment:
Same comment here about missing `IConfiguration` overloads. Here is the one I did a while back: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main-logs/src/OpenTelemetry/Logs/ExportLogRecordProcessorOptions.cs